script/build

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env node
+
+'use strict';
+
+let rollup = require('rollup').rollup;
+let babel = require('rollup-plugin-babel');
+let babelrc = require('babelrc-rollup').default;
+let filesize = require('file-size');
+let gzipSize = require('gzip-size').sync;
+let stat = require('fs').statSync;
+let minify = require('uglify-js').minify;
+let pkg = require('../package.json');
+
+rollup({
```

Comment:
I don't have a strong opinion, though I'd tend to incline towards using config files since it's a standard (and documented) way of using Rollup. I tend to use the `--environment` flag if I need to vary the configuration within a single config file. The duplication of work point is fair, though I wonder if we should just fix rollup/rollup#655 instead?

I do like the logging at the end of the build script, perhaps we should add that to the CLI